### DISCUSSION TITLE
Connection issue

### DISCUSSION THREAD

**Discussion by Ivanmecina** [akeeba/panopticon#801] [Q&A (Community Support)]
Hi I just updated to the latest version 1.2.2 and now I get a connection error in my Joomla sites. I think it is because of admintools .htaccess but I don't know how to fix it. I was thinking that seeing as both components are made by akeeba you should be able to help?

**Comment by nikosdion** [comment]
This does not make much sense. Updating Panopticon has nothing to do with Admin Tools.

**Reply by Ivanmecina** [reply]
The Joomla! API Application is blocked (HTTP error 403)

Something on your server - such as the server configuration controlled by your host, a server configuration file such as .htaccess, or a plugin of your site - is blocking Panopticon from accessing your site's API application (https://wiltshiremoney.org.uk/api).

Things to check:

Does your host, or your server configuration file (e.g. .htaccess), block access to the Joomla! API Application endpoint (https://wiltshiremoney.org.uk/api) altogether?
Does your host have a server–side protection solution (such as Apache's mod_security2) which might be blocking the Joomla! API Application endpoint (https://########/api), your API token (YOU_SHOULD_NEVER_POST_THIS), or the User-Agent of Panopticon (panopticon/1.2.2)?
Do you have a security, SEO, or other similar plugin which may be blocking the above?

**Reply by nikosdion** [reply]
Admin Tools does not block Joomla's API application.

You had posted your API token in this public discussion. Go to your user account and reset it.

I can see that I can access the API just fine. What cuts you off is CloudFlare.

In your CloudFlare account's Rules, Page Rules you need a rule with URL `wiltshiremoney.org.uk/api*` and setting Cache Level -> Bypass.

You also need a rule in Security, WAF, Custom Rules with "When incoming requests match…", URI path, wildcard, `/api*` AND IP source address, equals, the IP of your Panopticon instance – with "Then take action…" set to Skip and under "WAF components to skip" select all four checkboxes.

**Comment by Ivanmecina** [comment]
Hi I have tried to set these rules up and I still can’t access my site

these are the expressions I have set up

(http.request.full_uri contains "https://wiltshiremoney.org.uk/api*")

and

(http.request.uri.path wildcard r"/api*" and ip.src eq 78.142.47.19)
What am I doing wrong? please advise

Saludos Ivan

**Reply by nikosdion** [reply]
Well, nobody else has this problem, I can connect to the API application on your site just fine, and you get a message which says that Panopticon cannot. This means that either something on the server where Panopticon is installed blocks it, or something in front of or on the server your site is installed blocks it. None of that is under my control so that's as much as I can possibly help.
